pythran/config.py

```diff
@@ -157,7 +159,7 @@ def fmt_define(define):
                       distutils.sysconfig.get_config_var('VERSION'))
 
     if output:
-        print(' '.join(output))
+        print((' '.join(output)))
```

Comment:
> > -        print(' '.join(output))
> > +        print((' '.join(output)))
> 
> What a lot of parentheses
Yup, I'll do a cleanup pass before merging this commit, too many
conversion artefacts